Devscord.DiscordFramework/Framework/Commands/Properties/BotCommandPropertyType.cs

```diff
@@ -7,6 +7,7 @@ public enum BotCommandPropertyType
         Number = 0x4,
         Time = 0x8,
         UserMention = 0x16,
-        ChannelMention = 0x32
+        ChannelMention = 0x32,
```

Comment:
tak teraz myśle
bo zapis z 0x to chyba szesnastkowy XD

chyba trzeba będzie zrobić potężnego replace

Reply:
To w sumie dlaczego używamy szesnastkowego a nie dziesiętnego?